homeassistant/components/asuswrt/config_flow.py

```diff
@@ -42,28 +45,42 @@
     DEFAULT_INTERFACE,
     DEFAULT_TRACK_UNKNOWN,
     DOMAIN,
+    FLOW_VERSION,
     MODE_AP,
     MODE_ROUTER,
+    PROTOCOL_HTTP,
+    PROTOCOL_HTTPS,
     PROTOCOL_SSH,
     PROTOCOL_TELNET,
 )
 
-LABEL_MAC = "LABEL_MAC"
+ALLOWED_PROTOCOL = {
+    PROTOCOL_HTTPS: "HTTPS",
+    PROTOCOL_HTTP: "HTTP",
+    PROTOCOL_SSH: "SSH",
+    PROTOCOL_TELNET: "Telnet",
+}
```

Comment:
What's the reason it's a dict and not a list?
```python
ALLOWED_PROTOCOL = {
    "https": "HTTPS",
    "http": "HTTP",
    "ssh": "SSH",
    "telnet": "Telnet",
}
```

Reply:
Was just for label translation. Replaced the dict with a list and used a `selector` in the schema with `translation_key`